internal/cli/atlas/logs/download.go

```diff
@@ -89,12 +90,11 @@ func DownloadBuilder() *cobra.Command {
 		Long:  downloadLong,
 		Args:  require.ExactArgs(argsN),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			opts.initDefaultOut()
-			return opts.PreRunE(opts.ValidateProjectID, opts.initStore)
-		},
-		RunE: func(cmd *cobra.Command, args []string) error {
 			opts.host = args[0]
 			opts.name = args[1]
+			return opts.PreRunE(opts.ValidateProjectID, opts.initStore, opts.initDefaultOut)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
@andreaangiolillo the error is here, I was calling `initDefaultOut` before `opts.name = args[1]` had any value so it was being overwritten by the Run command

Reply:
oh got it, thanks for the clarification